dataset_config/platinum_genomes/gcs.json

```diff
@@ -0,0 +1,10 @@
+// Fields for GCS indexer
+{
+  "gcs_paths": [
+    // GCS path with PRIMARY_KEY. There is an implied .+ at the end.
+    // Example: gs://genomics-public-data/platinum-genomes/bam/PRIMARY_KEY_
+    // For more information about primary key, see
+    // https://github.com/DataBiosphere/data-explorer-indexers#overview
+    "gs://genomics-public-data/platinum-genomes/bam/PRIMARY_KEY_"
```

Comment:
I'm confused about what PRIMARY_KEY is doing here. Is this config asking me to specify my path as `gs://test-data-explorer/directory/path/PRIMARY_KEY_participant_id`?

Reply:
Expanded comments. Does it make sense now?

Reply:
Thanks, I see what you're doing now.

Under this design, we assume that the PRIMARY_KEY is present in the file path, and also consistently placed at the beginning of either a subdirectory or file name. Both are pretty heavy assumptions. For example, I couldn't use this indexer if my GCS objects looked something like `20180728_NA12877_S1.bam` where `NA12877` correlates to participant_ids. These assumptions might hold up for AMP-PD as it currently is, but are pretty restrictive beyond that. Have you considered any alternatives?

Reply:
As discussed offline, let's keep tabs on how to make these assumptions less restrictive and document for now that GCS filepaths must contain primary_keys in a uniform manner that map to those used by other sources (i.e. bigquery)

Reply:
Done